oas_docs/README.md

```diff
@@ -1,42 +1,61 @@
 # Kibana API reference documentation
 
-Documentation about our OpenAPI bundling workflow and configuration. See Kibana's hosted [stateful](https://www.elastic.co/docs/api/doc/kibana) and [serverless](https://www.elastic.co/docs/api/doc/serverless) docs.
+Documentation about our OpenAPI bundling workflow and configuration.
+The finalized OpenAPI documents are published to [Kibana API docs](https://www.elastic.co/docs/api/doc/kibana) and [Kibana Serverless API docs](https://www.elastic.co/docs/api/doc/serverless).
 
 ## Workflow
 
-The final goal of this workflow is to produce an OpenAPI bundle containing all Kibana's public APIs.
+This workflow produces OpenAPI documents that contain all of the Kibana public APIs.
+It runs automatically by using Buildkite scripts.
+You can also generate the documents by using `make` commands.
 
-### Step 0
+### Step 1 - Generate OpenAPI from code
 
-OAS from Kibana's APIs are continuously extracted and captured in `bundle.json` and `bundle.serverless.json` as fully formed OAS documentation. See `node scripts/capture_oas_snapshot --help` for more info.
+Kibana OpenAPI specifications are extracted from the code by `@kbn/capture-oas-snapshot-cli`.
+In particular, it captures information about HTTP APIs that use the following components:
 
-These bundles form the basis of our OpenAPI bundles to which we append and layer extra information before publishing.
+1. The Kibana Core `router` or `router.versioned` methods for defining HTTP APIs provided via the `core.http` service to all plugins
+1. Either `@kbn/config-schema` or `@kbn/zod` request and response schemas.
 
-### Step 1
+For more information about generating this snapshot, run `node scripts/capture_oas_snapshot --help`.
 
-Append pre-existing bundles not extracted from code using [`kbn-openapi-bundler`](../src/platform/packages/shared/kbn-openapi-bundler/README.md) to produce the final resulting bundles.
+Per [capture_oas_snapshot.sh](https://github.com/elastic/kibana/blob/main/.buildkite/scripts/steps/checks/capture_oas_snapshot.sh), OpenAPI snapshots are captured automatically for a specific list of endpoints.
+The results are stored in `oas_docs/bundle.json` and `oas_docs/bundle.serverless.json`.
 
-To add more files into the final bundle, edit the appropriate `oas_docs/scripts/merge*.js` files.
+NOTE: The `capture_oas_snapshot` script also validates the output files using `@kbn/validate-oas`.
+To see the validation results locally, run `node scripts/validate_oas_docs.js`.
 
-### Step 2
+### Step 2 - Merge more OpenAPI documents
 
-Apply any final overalys to the document that might include examples or final tweaks (see the ["Scripts"](#scripts) section for more details).
+Information about some APIs cannot be extracted from code.
```

Comment:
It'll be good to document why they can't be extracted to avoid similar patterns in the HTTP guidelines docs.